csharp/HOLMS.Platform/HOLMS.Platform/Support/Currency/DecimalDollars.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System;
+using HOLMS.Types.Primitive;
+
+namespace HOLMS.Platform.Support.Currency {
+    /// <summary>
+    /// DecimalDollars does high-precision monetary arithmetic, backed by the
+    /// CLR's System.Decimal. Internal calculations and db-backed model objects
+    /// should use this class.
+    /// </summary>
+    public struct DecimalDollars : IEquatable<DecimalDollars> {
```

Comment:
For sake of symmetry and completeness, I think `IComparable<DecimalDollars>` should be implemented here.